pyincore/analyses/transportationrecovery/transportationrecovery.py

```diff
@@ -1,304 +1,19 @@
-from __future__ import division
-import pandas as pd
-import copy
-import random
+# This program and the accompanying materials are made available under the
```

Comment:
Could you add the copyright year here. I recall this analysis is from 2018.

Reply:
Added and used as 2018

Reply:
Did you push? Sorry I don't see it gets reflected.